tgui/packages/tgui/interfaces/Keybinds/formatKeyboardEvent.ts

```diff
@@ -3,16 +3,16 @@ import { isEscape, KEY } from 'tgui-core/keys';
 const DOM_KEY_LOCATION_NUMPAD = 3;
 const KEY_CODE_TO_BYOND: Record<string, string> = {
   DEL: 'Delete',
-  DOWN: 'South',
+  ARROWDOWN: 'South',
```

Comment:
Is this correct?

Reply:
I think so. I wasn't able to bind arrow keys before the fix, after this fix I could bind them.

It's also what the specs says it should be: https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values#navigation_keys

Reply:
Weird. IIRC I got those values directly from pressing keys via Byond.

Reply:
I figured it out, it's not simply the browser's `KeyboardEvent`, it's a *React event*.

`export const formatKeyboardEvent = (event: React.KeyboardEvent): string => { ... };`

https://react.dev/reference/react-dom/components/common#react-event-object

>(React event objects) conforms to the same standard as the underlying DOM events, but fixes some browser inconsistencies.

Reply:
Also of note, this is formatKeyboardEvent.ts, made for the new Keybind.tsx window in #21681
It's not keys.ts, which is what I'm assuming you made and remains unchanged.
https://github.com/goonstation/goonstation/blob/09f7daa21458f602b33341c8de9b6a2b01a98a15/tgui/packages/common/keys.ts#L38